languages/js/src/errors.ts

```diff
@@ -35,12 +35,8 @@ export class DuplicateInstanceRegistrationError extends PolarError {
 }
 
 export class InlineQueryFailedError extends PolarError {
-  constructor(file?: string) {
-    if (file !== undefined) {
-      super(`Inline query failed while loading '${file}'.`);
-    } else {
-      super('Inline query failed.');
-    }
+  constructor(source: string) {
+    super(`Inline query failed: ${source}`);
```

Comment:
Did we remove the optional filename handling because `source` will contain the same info?

Reply:
Yeah. If it's from a file the location and filename are part of the message.